automatic/dotnet-uninstaller/dotnet-uninstaller.nuspec

```
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>dotnet-uninstaller</id>
+    <version>0.0.0</version>
+    <title>Microsoft .NET Core SDK Uninstall tool</title>
+    <authors>Microsoft</authors>
+    <owners>Swiss GRC AG</owners>
+    <projectUrl>https://learn.microsoft.com/en-us/dotnet/core/additional-tools/uninstall-tool</projectUrl>
+    <title>Microsoft .NET Core SDK Uninstall Tool</title>
+    <authors>Microsoft</authors>
+    <owners>Swiss GRC AG</owners>
+    <projectUrl>https://learn.microsoft.com/en-us/dotnet/core/additional-tools/uninstall-tool</projectUrl>
+    <projectSourceUrl>https://github.com/dotnet/cli-lab/releases</projectSourceUrl>
+    <packageSourceUrl>https://github.com/dotnet/cli-lab/tree/main/src</packageSourceUrl>
+    <licenseUrl>https://github.com/dotnet/cli-lab/blob/main/LICENSE.TXT</licenseUrl>
+    <releaseNotes>https://github.com/dotnet/cli-lab/releases</releaseNotes>
+    <docsUrl>https://learn.microsoft.com/en-us/dotnet/core/additional-tools/uninstall-tool?tabs=windows</docsUrl>
```

`?tabs=windows` is not required here

```suggestion
    <docsUrl>https://learn.microsoft.com/en-us/dotnet/core/additional-tools/uninstall-tool</docsUrl>
```